src/wpt/harness/harness.ts

```diff
@@ -737,6 +744,27 @@ globalThis.test = (func, name, properties): void => {
   }
 };
 
+/**
+ * Global version of :js:func:`Test.step_timeout` for use in single page tests.
+ *
+ * @param func - Function to run after the timeout
+ * @param timeout - Time in ms to wait before running the
+ * test step. The actual wait time is ``timeout`` x
+ * ``timeout_multiplier``.
+ */
+globalThis.step_timeout = (
+  func: UnknownFunc,
+  timeout: number,
+  ...rest: unknown[]
+): ReturnType<typeof setTimeout> => {
+  // eslint-disable-next-line @typescript-eslint/no-this-alias -- Copied as-is from WPT code
+  const outer_this = this;
+
+  return setTimeout(function () {
```

Comment:
Why don't we just make this an arrow function and get rid of outer_this value?

Reply:
I was just doing copy-pasta. There's a few other cases in WPT's harness like it; I can get rid of them all.

Reply:
Sounds good to me